src/main/python/main/ayab/preferences.py

```
@@ -138,9 +145,27 @@ def __init__(self, parent: GuiMain):
         self.refresh()
 
     def refresh(self) -> None:
+        """Sync variables and recent files list with saved configuration"""
         for var in self.variables.keys():
             self.settings.setValue(var, self.value(cast(PreferencesDictKeys, var)))
 
+        # Read recents from configuration
+        # and check if files are still available
+        for i in range(self.MAX_RECENT_COUNT):
+            filename = self.settings.value("Recent/" + str(i))
+            if filename != None and os.path.exists(filename) and not (filename in self.recentFiles):
```

Ruff (0.15.1)

[error] Comparison to `None` should be `cond is not None`

Replace with `cond is not None`

(E711)